src/test/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifierIntegrationTest.java

```diff
@@ -36,25 +43,34 @@
  */
 @RunWith(PowerMockRunner.class)
 @PrepareForTest({DisplayURLProvider.class, Office365ConnectorWebhookNotifier.class, Run.class})
-public class Office365ConnectorWebhookNotifierTestInt {
+public class Office365ConnectorWebhookNotifierIntegrationTest {
```

Comment:
renamed to standard test as `buildPlugin()` has problem with integration tests which are not executed as they should and it is not easy or possible to customize build command